service/app/util/Global.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * 
+ */
+package util;
+
+import java.lang.reflect.Method;
+import java.util.UUID;
+
+import org.sunbird.actor.service.SunbirdMWService;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.common.request.ExecutionContext;
+import org.sunbird.common.responsecode.ResponseCode;
+
+import controllers.BaseController;
+import play.Application;
+import play.GlobalSettings;
+import play.libs.F.Promise;
+import play.libs.Json;
+import play.mvc.Action;
+import play.mvc.Http;
+import play.mvc.Http.Request;
+import play.mvc.Result;
+import play.mvc.Results;
+
+/**
+ * This class will work as a filter.
+ * 
+ * @author Manzarul
+ *
+ */
+public class Global extends GlobalSettings {

```

Comment:
Create a reusable GlobalBase in a common-service project which can be added as dependency by any sunbird micro-service.

Reply:
Done